examples/minimal/package.json

```diff
@@ -31,6 +31,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.11.2",
+    "@graphql-modules/core": "^0.7.17",
```

Comment:
This should be part of Npm.depends i guess

Reply:
Great attention to details as always.